README.md

```diff
@@ -123,6 +123,19 @@ cp /usr/i686-w64-mingw32/bin/lib{gcc_s_dw2-1,ssp-0,stdc++-6,winpthread-1}.dll /m
 wine /my/path/to/Winamp/winamp.exe
 ```
 
+## Building & Running on Windows
+
+Don't. But if you must there are to ways to build
+
+1. open the folder with Visual Studio
+2. use the generate_project_files.bat script, which produces a 32-bit only Visual Studio project using CMake.
```

Comment:
Either:

1. just put the commands right here, the script is basically the same as the linux versions.
    ```cmd
    del /s /q .\build
    md build
    pushd build
    cmake -DCMAKE_GENERATOR_PLATFORM=Win32 ..\
    popd
    ```
1. or just skip the whole cmdline thing, if importing in VS is that comfortable.

i'm in favor of 2 for now.